RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,119 @@
+---
+rpip: #<to be assigned>
+title: Bounty Standards
+description: Describes standards and incentives for the production of effective bounty proposals.
+author: LongForWisdom (@LongForWisdom)
+discussions-to: <URL>
+status: Draft
+type: Meta
+created: 2024-01-04
+requires: 26
+---
+
+## Abstract
+
+TODO Write
+
+## Motivation
+
+TODO Write
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119. TODO Remove
+
+### Definitions
+
+**Bounty Proposal**  
+
+A high-level bounty idea that requires more work before it can be adopted by the GMC and made available to bounty hunters. The goal of the bounty proposal is to convince the GMC that this idea is worth the time and effort to adminster. Requirements for proposals are comparitively easy to meet, in order to encourage the community to submit their ideas. Bounty proposals are the 'quick and dirty' path for community members to suggest a bounty idea to the GMC.
+
+**Bounty Definition**  
+
+A detailed and well defined bounty specification that can be adopted by the GMC and made available to bounty hunters. The goal of the bounty definition is to ensure that the bounty deliverables are presented to bounty hunters in a way that results in accurate and effective output taking place under terms fully understood by both parties. Bounty definitions may be delivered by community members alongside their bounty proposal or may be created by the GMC. Bounty definitions are the 'tick all the boxes' path for community members to get their bounty adopted by the GMC.
+
+**Bounty Hunter**  
+
+A person or group that works to deliver the work specified by a well defined bounty.
+
+**Bounty Contact**  
+
+An individual that has expressed willingness to act as a point of contact with bounty hunters for a given bounty. This may be the bounty proposal author, a member of the GMC, or another Rocket Pool community member. The goal of the bounty contacts is to support and liase with bounty hunters so that they are better able to deliver the work specified in the bounty definition. Bounty contacts are listed in bounty definitions. 
+
+### Bounty Lifecycle
+![image](../assets/rpip-bounty_standards/bounty-lifecycle.png)
+
+#### Statuses
+**Draft - Proposed** - The bounty has been described and proposed to the GMC for adoption. The bounty is not yet eligible for adoption.  
```

Comment:
Was wondering about the Draft statuses. Essentially, should all proposals aim to be "defined" and the GMC decides if there’s anything they would like to change to give the proposal a better shot at passing before final debate and vote?

I just think most people will think their idea is complete and someone will have to validate them anyway so it might save some strife if the expectation is a complete proposal that the GMC might consent ask to modify to give it a better chance of passing. Think it works as described in your draft, but shifts the burden of determining proposed/defined status to the RPIP editors which may not fit their work flow as well

Reply:
I think the expectation is that the GMC will be validating the definitions if they don't write them.